tsfile/src/main/java/org/apache/iotdb/tsfile/write/writer/TsFileIOWriter.java

```diff
@@ -277,6 +275,10 @@ public void endFile(Schema schema) throws IOException {
     int size = tsFileMetaData.serializeTo(out.wrapAsStream());
     logger.debug("finish flushing the footer {}, file pos:{}", tsFileMetaData, out.getPosition());
 
+    // write bloom filter
+    size += tsFileMetaData.serializeBloomFilter(out.wrapAsStream(), chunkGroupMetaDataList);
+    logger.debug("finish flushing the bloom filter file pos:{}", out.getPosition());
```

Comment:
Wrap with `if (logger.isDebugEnabled())`, `out.getPosition()` could be costly, please also fix logs above.